Gemfile.lock

```diff
@@ -252,7 +250,7 @@ GEM
     rb-fsevent (0.10.4)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
-    regexp_parser (2.0.3)
+    regexp_parser (2.1.1)
```

Comment:
### [regexp_parser](https://github.com/ammar/regexp_parser)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.0.3 → 2.1.1

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **19** commits. See the full changes on [the compare page](https://github.com/ammar/regexp_parser/compare/v2.0.3...v2.1.1).

These are the first 10 commits:
- (9ae057a) [Add basic rubocop linting](https://github.com/ammar/regexp_parser/commit/9ae057abe5bf2d9c19aa164ba888d960c4068d77)
- (c50a0fc) [Prefer __dir__ or require_relative over __FILE__](https://github.com/ammar/regexp_parser/commit/c50a0fc0b7c5218bedfde1918a3f609b8b6e0f95)
- (4729bae) [Fix scanning of \g<0>, \g'0' after f2e700b](https://github.com/ammar/regexp_parser/commit/4729bae297d66bfce6c0683d4858c7cb42753106)
- (d27abf1) [Fix scanning of some exotic ref/call ids](https://github.com/ammar/regexp_parser/commit/d27abf13fb0435220f2d6a9161673e917805b794)
- (9fd8381) [Update dev dep](https://github.com/ammar/regexp_parser/commit/9fd8381268d060f62741cafa44ca6449b1b1b26d)
- (6c331d0) [Clean up Parser a bit ...](https://github.com/ammar/regexp_parser/commit/6c331d01dcdbdffe3c57c2d1982f5731a37722e5)
- (e34d653) [Simplify lookup](https://github.com/ammar/regexp_parser/commit/e34d6535a0c4f95a9b8128e57b5e2f4d21bcf986)
- (f90c8df) [Spacing](https://github.com/ammar/regexp_parser/commit/f90c8df17e1471e893699341f74fe24fb3f8e002)
- (0017a1d) [Typo](https://github.com/ammar/regexp_parser/commit/0017a1dbfb79ca7e35fb364e01b97dc36eab7f78)
- (4151764) [Support meta/control escapes within sets](https://github.com/ammar/regexp_parser/commit/41517647fd31c3ef51eb572bb3731ff72bd24fda)


</details>